libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -256,7 +262,8 @@ bool AP_BattMonitor::has_current(uint8_t instance) const
         return (_monitoring[instance] == BattMonitor_TYPE_ANALOG_VOLTAGE_AND_CURRENT ||
                 _monitoring[instance] == BattMonitor_TYPE_SOLO ||
                 _monitoring[instance] == BattMonitor_TYPE_BEBOP ||
-                _monitoring[instance] == BattMonitor_TYPE_MAXELL);
+                _monitoring[instance] == BattMonitor_TYPE_MAXELL||
```

Comment:
I hadn't noticed this before, but can you add a space before the OR operator here?